avidemux_sk.ts

```diff
@@ -3385,12 +3385,12 @@ Skúsiť to znovu?</translation>
     <message>
         <location filename="../../../avidemux_core/ADM_core/src/ADM_prettyPrint.cpp" line="+23"/>
         <source>%d minute(s)</source>
-        <translation>%d minút</translation>
+        <translation>%n minút</translation>
```

Comment:
If merged, this translation, which reverts the fix I commited on Dec 20, 2021, will make Avidemux crash. Format specifiers like `%d`, `%u`, `%x`, `%s` must be preserved by translations, they are not interchangeable.

We cannot use numerus forms in source strings marked for translation by QT_TRANSLATE_NOOP macro, this is a limitation of the way how translation works in Avidemux. In a few cases where we do use numerus forms, we have to circumvent this infrastructure by using `QCoreApplication::translate()` directly, available only for Qt code (i.e. not in core).